gym_ai2thor/envs/ai2thor_env.py

```diff
@@ -156,8 +172,22 @@ def step(self, action, verbose=True):
                     inventory_changed_str = ''
                 print('{}: {}. {}'.format(
                     action_str, interaction_obj['objectType'], inventory_changed_str))
+        elif action_str.startswith('Rotate'):
+            if self.continuous_movement:
+                # Rotate actions
+                if action_str.endswith('Left'):
+                    self.absolute_rotation -= self.rotation_amount
+                    self.event = self.controller.step(
+                        dict(action='Rotate', rotation=self.absolute_rotation))
+                elif action_str.endswith('Right'):
+                    self.absolute_rotation += self.rotation_amount
+                    self.event = self.controller.step(
```

Comment:
1 indentation less to this line, since it is applicable for both left and right. Then you can remove lines 180 and 181